### DISCUSSION TITLE
kf.setImage broken in 6.2.0?

### DISCUSSION THREAD

**Discussion by Jsayer7** [onevcat/Kingfisher#1651] [General]
Is anyone else having issues getting images to download and display after today's upgrade to version to 6.2.0? I have a pretty simple app that just gets images from a URL and the imageView (using UIKit) just shows the progress indicator spinner indefinitely.

**Comment by onevcat** [comment]
Maybe it is an unexpected regression. I’ll check it soon!

**Comment by onevcat** [comment]
@Jsayer7 

There is an internal CFNetwork error in 6.2.0. It is a regression and 6.2.1 is released. Can you try the latest version to see whether it fixes your issues?

Thanks!

**Reply by Jsayer7** [reply]
I've confirmed that 6.2.1 works. Updated packages and the setImage function now sets the image just fine. 

Thanks!
